.github/workflows/gempyor-ci.yml

```diff
@@ -49,6 +49,7 @@ jobs:
         shell: bash
       - name: Run gempyor tests
         run: |
+         export FLEPI_PATH=$(pwd)
```

Comment:
is there a whitespace difference here? i assume it doesn't matter syntactically, but should dress it (unless it matters to making syntax correct).

Reply:
Fixed